### PR TITLE
BBH executable: Observe Ah in inertial frame

### PR DESCRIPTION
**Pull request by markscheel**
## Proposed changes
    
The Ah is computed in the grid frame and observed in the inertial frame, so quantities are interpolated in both frames.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Comment by markscheel** [comment]
The failing tests are timeouts of tests that are unrelated to and unaffected by this PR.

File: src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp [hunk 1]
```diff
@@ -180,31 +180,29 @@ struct EvolutionMetavars {
...
-                 StrahlkorperTags::InvJacobianCompute<::Frame::Grid>,
-                 StrahlkorperTags::DxRadiusCompute<::Frame::Grid>,
-                 StrahlkorperTags::OneOverOneFormMagnitudeCompute<
-                     volume_dim, ::Frame::Grid, DataVector>,
-                 StrahlkorperTags::NormalOneFormCompute<::Frame::Grid>,
-                 StrahlkorperTags::UnitNormalOneFormCompute<::Frame::Grid>,
-                 StrahlkorperTags::UnitNormalVectorCompute<::Frame::Grid>,
-                 StrahlkorperTags::GradUnitNormalOneFormCompute<::Frame::Grid>,
-                 StrahlkorperTags::ExtrinsicCurvatureCompute<::Frame::Grid>,
-                 StrahlkorperGr::Tags::SpinFunctionCompute<::Frame::Grid>>,
```
Comment (geoffrey4444) [inline src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp]: Why did these tags change? E.g. no spin function anymore in the horizon compute items on target...is it because the missing ones were not actually used in observers?

File: src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp [hunk 2]
```diff
@@ -180,31 +180,29 @@ struct EvolutionMetavars {
...
       gr::Tags::SpatialMetric<volume_dim, ::Frame::Grid, DataVector>,
       gr::Tags::InverseSpatialMetric<volume_dim, ::Frame::Grid>,
       gr::Tags::ExtrinsicCurvature<volume_dim, ::Frame::Grid>,
-      gr::Tags::SpatialChristoffelSecondKind<volume_dim, ::Frame::Grid>>;
-  // Observe both horizons in grid frame too.
-  using horizons_tags_to_observe =
-      tmpl::list<StrahlkorperGr::Tags::AreaCompute<::Frame::Grid>,
-                 StrahlkorperGr::Tags::IrreducibleMassCompute<::Frame::Grid>>;
+      gr::Tags::SpatialChristoffelSecondKind<volume_dim, ::Frame::Grid>,
+      gr::Tags::SpatialMetric<volume_dim, ::Frame::Inertial, DataVector>>;
```
Comment (geoffrey4444) [inline src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp]: Why did you add SpatialMetric here?

**Review by geoffrey4444 — DISMISSED** [review]
Happy to approve. I have a couple of questions for my own understanding. 

I tested this, and it works in the sense that I get reasonable-looking output when running a head-on BBH evolution. The code looks reasonable to me.

One suggestion for a future PR: either @markscheel or I should probably add an AhC horizon finder that only triggers once the black holes are about to merge. But that's beyond the scope of this PR

File: src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp (hunk 1 above)
Comment (markscheel) [reply src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp]: Yes. There were multiple tags that were not being used, so I removed them.  If we want to observe more quantities, then we can add more tags here.

File: src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp (hunk 2 above)
Comment (markscheel) [reply src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp]: I added SpatialMetric because the observers depend on it.

Note that the SpatialMetric I added here is in the Inertial frame, whereas the one a few lines above is in the Grid frame.  The AH finder (which runs in the grid frame) needs SpatialMetric in the Grid frame, and the observers (which are computed in the inertial frame) also need SpatialMetric but in the Inertial frame.  So SpatialMetric in both frames are interpolated to the AH.

**Review by nilsdeppe — DISMISSED** [review]

**Comment by kidder** [comment]
@markscheel this needs a rebase

**Comment by markscheel** [comment]
Rebased and fixed conflicts.

**Review by knelli2 — APPROVED** [review]
We should merge this soon as more changes to the BBH executable will probably be coming in the near future (control systems, etc...)

**Review by kidder — APPROVED** [review]
